pkg/subscriptions.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Broke this out to give focus on topic subscriptions used on startup @restevens402 welcome feedback or ways to refactor